src/less/modules/form-component.less

```diff
@@ -115,6 +115,8 @@
 			}
 		}
 		textarea {
+			max-height: 130px;
```

Comment:
can you try and make this configurable?

Reply:
try using a user specified class or property for textarea to make this css effect